indexer/cmd/indexer/cli.go

```diff
@@ -19,6 +19,12 @@ var (
 		Usage:   "path to config file",
 		EnvVars: []string{"INDEXER_CONFIG"},
 	}
+	MigrationsFlag = &cli.StringFlag{
```

Comment:
Do we need to make this configurable? ffti but I think it's fine to simply hardcode the "./migrations" path since the user shouldn't ever be altering those files anyways

Reply:
It's tough because the migrations file path is different depending on if you are running this locally vs running this in a docker container. I feel like there is a better way to handle it though without making it configurable 
![image](https://github.com/ethereum-optimism/optimism/assets/35039927/4ec50f3c-e1d0-4e94-bdc9-b4e2081305f6)

Reply:
sounds good, lets leave it as in then